metricbeat/module/system/process_summary/_meta/data.json

```diff
@@ -15,14 +15,10 @@
     "system": {
         "process": {
             "summary": {
-                "dead": 0,
-                "idle": 63,
-                "running": 3,
-                "sleeping": 117,
-                "stopped": 0,
-                "total": 185,
-                "unknown": 2,
-                "zombie": 0
+                "idle": 122,
```

Comment:
I see a similar YML file in the same directory, does it need to be updated?
 
https://github.com/elastic/beats/blob/main/metricbeat/module/system/process_summary/_meta/fields.yml

Reply:
Similarly, is there a changelog entry or any documentation updates that need to happen as part of these changes?

Reply:
Ah, I knew I was forgetting something.